### PR TITLE
edits

### PR DESCRIPTION
**Pull request by jsalvatier**
This reverts commit 617513ee14da31165b0650c880c7cf412e685564.

### CODE REVIEW

File: reference/UQ Handbook Chapter.ipynb [hunk 1]
```diff
@@ -20,7 +20,7 @@
    "cell_type": "markdown",
```
Comment (jsalvatier) [inline reference/UQ Handbook Chapter.ipynb:20]: Can't quite comment in the right spots.

Are the readers going to know what MCMC is? The first couple of sentences seem to be written as if the readers will be fairly familiar with it.

Comment (fonnesbeck) [inline reference/UQ Handbook Chapter.ipynb:20]: I would have thought so. I can easily add some MCMC background, but I wasn't clear how much of an MCMC tutorial it should be, as opposed to an intro to PyMC. I envisioned the target audience as being someone familiar with the methods, but not the software.

File: reference/UQ Handbook Chapter.ipynb [hunk 2]
```diff
@@ -91,12 +91,7 @@
     "alpha, beta = 1, 1\n",
     "\n",
     "# Simulate outcome variable\n",
-    "Y = alpha + X*beta + np.random.randn(size)\n",
-    "\n",
-    "# Define helper function\n",
-    "def print_public(module):\n",
-    "    import pprint\n",
-    "    pprint.pprint([name for name in dir(module) if name[0].istitle()])"
+    "Y = alpha + X*beta + np.random.randn(size)"
```
Comment (jsalvatier) [inline reference/UQ Handbook Chapter.ipynb:94]: I wonder if we shouldn't call X and Y Xdata and Ydata to make it clearer that they're just arrays.

**Comment by jsalvatier** [comment]
Also, I wonder if it doesn't make sense to discuss simple model fitting (at least MAP) before launching into another way to write the same model. Or perhaps this was an intentional choice to first give several examples of specifying models? i'm worried that the readers haven't yet seen the payoff of actually getting some kind of answer.

**Comment by jsalvatier** [comment]
I'll do more reviewing tomorrow.

**Comment by twiecki** [comment]
Can we just merge this?

File: reference/UQ Handbook Chapter.ipynb (hunk 1 above)
Comment (jsalvatier) [inline reference/UQ Handbook Chapter.ipynb:20]: Yes, please go ahead and add some MCMC background. Maybe say "skip to section X if you're already familiar with MCMC" or something to that effect.

**Pull request merged**